### ISSUE TITLE
EFI issue on identical machine - black screen after OS picker

### ISSUE THREAD

**Issue by hitchhiker90** [blacksd/Dell-Inspiron-7559-hackintosh-opencore#1]
Hello,

I have an identical laptop to yours and was [recommended your repo](https://www.insanelymac.com/forum/topic/352042-need-help-with-dell-inspiron-7559-opencore-hangs-after-picking-installation-option/), however it does not work on my machine (basically I am running into the same issue as with my original config). I have tried with OS versions from Monterey to Catalina, and with various tweaks found online, and it seems that nothing makes no difference at this point.

I'm running out of ideas for what could be the issue, and since your config is basically identical, I wanted to ask what BIOS version you were using and if you have made any meaningful changes in the BIOS settings (particularly if they are not mentioned in the OC guide)?

I hope this is not too much trouble, of course feel free to ignore/delete this.

Thanks and best regards.

**Comment by blacksd** [comment]
Hi, that's absolutely a fair point - I remember having some issues with the most up-to-date BIOS version and some of the settings. Let me double-check my settings and I'll make a PR with some added instructions.

Don't lose hope - I promise this machine will turn up a pretty good hackintosh (been my daily driver for 1+ year!)

**Comment by hitchhiker90** [comment]
Hi again, I was able to get past this particular issue, turns out I did miss a BIOS setting, the one listed as **Intel Platform Trust** in the guide, which in my BIOS is called **Firmware TPM**. It's actually not that difficult to find the association between the two (it's enough to google 'Inter Platform Trust'), so I should've been just that little bit more thorough with that part of the guide (it's also more important in this particular case than the guide makes it out to be).

Now I can get the installer to start using your EFI. My original files still result in some errors, but now I am getting output to the display + I know that the issue is with the EFI config itself, not a BIOS/hardware issue, and should be able to resolve this by referring to yours. Thanks a lot for sharing this!
